lib/process-services-cloud/src/lib/task/task-list/components/task-list-cloud.component.spec.ts

```diff
@@ -587,7 +577,7 @@ describe('TaskListCloudComponent', () => {
         // TODO: highly unstable test
         // eslint-disable-next-line
         xit('should show tooltip if config copyContent flag is true', fakeAsync(() => {
```

Comment:
do we really need this test if it's excluded?
can we double check if it's flaky, and try to use async (keyword not function) instead of fakeAsync ?

Reply:
Not my test, I only changed `fakeGlobalTask` -> `fakeGlobalTasks`